.github/workflows/autobuild.yml

```diff
@@ -213,29 +206,23 @@ jobs:
             /opt/android/android-ndk
           key:                      ${{ matrix.config.target_os }}-${{ hashFiles('.github/workflows/autobuild.yml', 'autobuild/android/autobuild_apk_1_prepare.sh', 'autobuild/android/install-qt.sh') }}-${{ matrix.config.cmd1_prebuild }}
 
-      # Prepare (install QT & dependencies)
-      - name:                       "Prepare for ${{ matrix.config.config_name }}"
-        if:                         ${{ matrix.config.cmd1_prebuild }}
+      - name:                       Download and install build dependencies for ${{ matrix.config.config_name }}
```

Comment:
Do we need to say "Download and"? Downloading is probably implicit.

Reply:
I had added that as a response to feedback by @ann0see.
@ann0see Should we keep "Download and" in or should I shorten it again?

Reply:
Maybe use Set up? But that would be close to the unwanted „Prepare“ again.

Reply:
Ok, done.